cypress/integration/list.js

```diff
@@ -171,16 +171,50 @@ describe('List Page', () => {
 
     describe('rowClick', () => {
         it('should accept a function', () => {
-            cy.contains('Fusce massa lorem, pulvinar a posuere ut, accumsan ac nisi').parents('tr').click();
+            cy.contains(
+                'Fusce massa lorem, pulvinar a posuere ut, accumsan ac nisi'
+            )
+                .parents('tr')
+                .click();
             cy.contains('Summary').should(el => expect(el).to.exist);
         });
 
         it('should accept a function returning a promise', () => {
             LoginPage.navigate();
             LoginPage.login('user', 'password');
             ListPageUsers.navigate();
-            cy.contains('Annamarie Mayer').parents('tr').click();
+            cy.contains('Annamarie Mayer')
+                .parents('tr')
+                .click();
             cy.contains('Summary').should(el => expect(el).to.exist);
         });
     });
+
+    describe('expand panel', () => {
+        it('should show an expand button opening the expand element', () => {
+            cy.contains('1-10 of 13'); // wait for data
```

Comment:
You should add an expectation here: `.should(el => expect(el).to.exist)`

Reply:
I don't want to do an assertion here, just wait. Also, the other tests in this file don't do that either.

Reply:
I've done some reading. 

The Cypress docs says you're wrong:

https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#For-instance

>**Core Concept**
>
>All DOM based commands automatically wait for their elements to exist in the DOM.
>
>You never need to write .should('exist') after a DOM based command.

And it repeats it:

https://docs.cypress.io/guides/references/best-practices.html#Unnecessary-wait-for-cy-get

And it repeats it again:

https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Commands-Are-Asynchronous

I only found one person saying what you say:

* https://stackoverflow.com/questions/53428220/cypress-io-contains-not-waiting-for-element

I am very unsure that you're right. If you can reproduce the `contains` not waiting before executing the next cypress command behavior, I suggest you open an issue in the Cypress bug tracker, because that's a bug of theirs.

Until then, I'll assume Cypress does what their doc says.

Reply:
I admit I haven't tested it